spanner/transaction.go

```diff
@@ -188,6 +192,9 @@ func (ro ReadOptions) merge(opts ReadOptions) ReadOptions {
 	if opts.RequestTag != "" {
 		merged.RequestTag = opts.RequestTag
 	}
+	if opts.DataBoostEnabled {
```

Comment:
Hmmm.... This means that you cannot override a `false` value in `ro`. I guess that is OK, as we don't allow 'unsetting' any of the other values through this method either. (No action needed, just thinking out loud....)